content/soarian/dstu2/medications/medication-statement.md

```diff
@@ -0,0 +1,83 @@
+---
+title: MedicationStatement | Soarian DSTU 2 API
```

Comment:
May want to add a comment about usage of contained. Also, likely need to change the binding table since it doesn't look like medicationCodeableConcept can ever be returned and make this a contained binding? 
http://fhir.cerner.com/millennium/dstu2/medications/medication-statement/#contained-medication-bindings